### PR TITLE
Use timeanddate.com for clickable times

### PR DESCRIPTION
**Pull request by joshmoore**
e.g. first time slot now takes you to https://www.timeanddate.com/worldclock/fixedtime.html?p1=3903&iso=2021-06-08T10:30:00

### CODE REVIEW

**Comment by jburel** [comment]
staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2021/day1/

Work nicely

**Comment by joshmoore** [comment]
Updated at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2021/day1/ with UTC:
Screen Shot 2021-06-07 at 13 24 07

**Comment by jburel** [comment]
Last commit makes sense and will avoid confusion.

File: _includes/ome2021_program_agenda.html [hunk 1]
```diff
@@ -19,7 +19,8 @@ <h4>Schedule</h4>
 <p>
     The content will repeated over two sessions to accomodate our
     community across timezones.<br/>
-    All times are in UK - British Summer Time (GMT+1).
+    All times are in UK - British Summer Time (GMT+1). Click on the
```
Comment (sbesson) [inline _includes/ome2021_program_agenda.html:22]:
```suggestion
    All times are in UK - British Summer Time (UTC+1). Click on the
```

Proposing to unify with the time format below and remove all mentions of GMT

Comment (joshmoore) [reply _includes/ome2021_program_agenda.html:22]: :+1:

**Pull request merged**